### ISSUE TITLE
Text changes and one minor spacing issue: Collection settings page

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#601]
1. Change 'Release and visibility', ' Terms of use and licenses', 'Collection participants', and 'Review workflow' to H3s.
2. Add a little more white space between the breadcrumbs and the collection details tabs. In my mockups I have about 50 pixels between them.

**Comment by astridu** [comment]
The spacing looks good. The text sizes look good, but they should be tagged as H2's.
